### ISSUE TITLE
maintaining this project

### ISSUE THREAD

**Issue by guysopher** [chirag200666/waterlock-local-sms-auth#2]
Hey,
Are you still maintaining this project?
It looks like something that can be useful...

**Comment by chirag200666** [comment]
@guysopher 
Not exactly, I made this for a project I was working on earlier, no longer working on it.
Do you want any specific feature requirement ?
Are you interested in maintaining the project ?
